### PR TITLE
fix: [CDC-16646]: Displayed Summary and Description fields on Jira Upd…

### PR DESCRIPTION
**Pull request by goyalsonali**
Summary:

Displayed summary and description fields on Jira Update 

Jira Links:

https://harness.atlassian.net/browse/CDC-16646

Screenshots:

https://user-images.githubusercontent.com/98508399/152391746-a8428308-b833-4e19-954c-0727c4e0f9f4.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by goyalsonali** [comment]
trigger cypress

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraDynamicFieldsSelector.tsx [hunk 1]
```diff
@@ -89,12 +92,21 @@ const SelectFieldList = (props: JiraDynamicFieldsSelectorContentInterface) => {
...
       const fieldKeys = Object.keys(issueTypeData?.fields || {})
-      fieldKeys.sort().forEach(keyy => {
-        if (issueTypeData?.fields[keyy] && keyy !== 'Summary' && keyy !== 'Description') {
-          fieldListToSet.push(issueTypeData?.fields[keyy])
-        }
-      })
-      setFieldList(fieldListToSet)
+      if (jiraType === 'createMode') {
+        fieldKeys.sort().forEach(keyy => {
+          if (issueTypeData?.fields[keyy] && keyy !== 'Summary' && keyy !== 'Description') {
```
Comment (chiragp1212) [inline src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraDynamicFieldsSelector.tsx:97]: Can we work towards removing this code duplication?

Comment (goyalsonali) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraDynamicFieldsSelector.tsx:97]: Yes, I will see into it.

**Comment by goyalsonali** [comment]
trigger cypress

**Comment by goyalsonali** [comment]
trigger rebase

**Comment by goyalsonali** [comment]
trigger cypress

**Comment by goyalsonali** [comment]
trigger cypress

**Comment by goyalsonali** [comment]
retrigger jest

**Comment by goyalsonali** [comment]
trigger cypress

**Pull request closed without merging**